layouts/section/events.html

```diff
@@ -14,18 +16,18 @@
       <!-- Content -->
       {{ partial "content/_index.html" . }}
       <!-- Upcoming Events -->
-      {{ if (len $upcomingEvents) "ge" 1 }}
+      {{ if (len $upcomingEventsMeetup) "ge" 1 }}
         <div class="content mt-2">
           <h2>⏳️
-            {{ len $upcomingEvents }}
+            {{ len $upcomingEventsMeetup }}
             Upcoming
-            {{ if gt (len $upcomingEvents) 1 }}Events{{ else }}Event
+            {{ if gt (len $upcomingEventsMeetup) 1 }}Events{{ else }}Event
             {{ end }}
           </h2>
         </div>
         <div class="columns is-multiline">
-          {{ range $upcomingEvents.ByDate.Reverse }}
-            {{ partial "content/child.html" . }}
+          {{ range $upcomingEventsMeetup }}
```

Comment:
We wouldn't want to replace all events with just the ones from Meetup - we would want to merge these somehow.

Reply:
so then you just want like events you already have written down to be replaced or something? 
or to add on the events already meetup events on top? as well as if we were to add this: issue #354 how would that work?

Reply:
or do you want a sperate section for each?

Reply:
@fvcproductions updated the pr with a separate section.
how do you check the deployment preview from netlify?